ooniprobe/AppDelegate.mm

```diff
@@ -122,7 +121,7 @@ - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)butto
     }
     else if (buttonIndex == 1 && alertView.tag == alert_tag_open_itunes){
         //open ooniprobe on iTunes connect
-        NSString *iTunesLink = @"itms://itunes.apple.com/us/app/apple-store/id1245670385?mt=8";
+        NSString *iTunesLink = @"itms://itunes.apple.com/us/app/apple-store/id1199566366?mt=8";
```

Comment:
What is the `mt=8` at the end for?

Reply:
https://stackoverflow.com/questions/1781427/what-is-mt-8-in-itunes-links-for-the-appstore